packages/evergreen-file-picker/test/index.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react'
+import test from 'ava'
+import render from 'react-test-renderer'
+import { shallow } from 'enzyme'
+import { FilePicker, CLASS_PREFIX } from '../src'
+
+test('snapshot', t => {
+  const component = <FilePicker />
+  const tree = render.create(component).toJSON()
+  t.snapshot(tree)
+})
+
+test('sets name', t => {
+  const component = shallow(<FilePicker name="hi" />)
+  t.is(component.find(`.${CLASS_PREFIX}-file-input`).prop('name'), 'hi')
+})
+
+test('sets accept', t => {
+  const component = shallow(<FilePicker accept="application/json" />)
+  t.is(
+    component.find(`.${CLASS_PREFIX}-file-input`).prop('accept'),
+    'application/json'
+  )
+})
+
+test('sets required', t => {
+  const component = shallow(<FilePicker required />)
+  t.is(component.find(`.${CLASS_PREFIX}-file-input`).prop('required'), true)
```

Comment:
Use `t.true`/`t.false` instead of `t.is(something, true/false)`.

Reply:
Opps, good point.